CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+### citus v10.2.4 (February 1, 2022) ###
+
+* Adds support for operator class parameters in indexes
+
+* Fixes a build error that happens when `lz4` is not installed
+
+* Fixes function signature generation bug
```

Comment:
maybe a more descriptive entry for that ?

Reply:
How about

```md
* Fixes a bug with distributed functions that have `OUT` parameters or
  return `TABLE`
```